### DISCUSSION TITLE
Add Reporting from using Pre-Existing List/Log

### DISCUSSION THREAD

**Discussion by lloyddunamis** [ThioJoe/YT-Spammer-Purge#569] [Ideas]
Option involved: `7. Remove comments using a pre-existing list or log file`

Just as it would switch to report-only when in "Not Your Channel Mode",
I wish for an option to `3. Report` spams when using pre-existing logs.

As of 2.15.0 Beta-2, it's just Delete or Hide.
![image](https://user-images.githubusercontent.com/6012867/152651437-9cdabdb0-0381-4412-955d-5dcc692d447b.png)

**Comment by lloyddunamis** [comment]
On the side, I was hoping the pre-existing list would be something of a workaround on the timeout issue on long idle (#567) for now, and a resumption from the already-scanned list of spams (#391).
It is, but only when the scanned spams are under my own channel. (and personally, I'd prefer `Report`ing spams first anyway, regardless if it's my own channel or somewhere else)

**Comment by ThioJoe** [comment]
The only thing to consider is if that was possible, it would effectively allow someone to just go and report any list, no matter the mode that was used to scan. And the reason only certain modes are allowed when not scanning your own channel is to prevent abuse
